.vscode/launch.json

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
file should be removed, there is no application in the plugin root. Application is in the `sample` folder. Simply open the sample directory in VS code and you have debugging there.